amundsen_application/api/utils/metadata_utils.py

```diff
@@ -23,17 +25,43 @@ def marshall_table_partial(table_dict: Dict) -> Dict:
     results = schema.dump(table).data
     # TODO: fix popular tables to provide these? remove if we're not using them?
     # TODO: Add the 'key' or 'id' to the base PopularTableSchema
-    results['key'] = f'{table.database}://{table.cluster}.{table.schema}/{ table.name}'
+    results['key'] = f'{table.database}://{table.cluster}.{table.schema}/{table.name}'
     results['last_updated_timestamp'] = None
     results['type'] = 'table'
 
     return results
 
 
+def parse_editable_rule(rule: MatchRuleObject,
+                        schema: str,
+                        table: str) -> bool:
+    """
+    Matches table name and schema with corresponding regex in matching rule
```

Comment:
nit: Can you add descriptions for the parameters and return values like we do elsewhere? This may seem straightforward but understanding exactly what the parameters are or where they come from, or what the boolean returned by the method represents will provide useful context in the future for newcomers.